src/routes/transactions/mappers/common/twap-order.mapper.spec.ts

```diff
@@ -40,7 +35,7 @@ describe('TwapOrderMapper', () => {
 
   beforeEach(() => {
     const multiSendDecoder = new MultiSendDecoder();
-    const gpv2Decoder = new GPv2Decoder(mockLoggingService);
```

Comment:
I've removed logging as I have a forthcoming refactor of the `AbiDecoder` that will simplify _all_ interactions with it (and include logging).

Reply:
I've implemented the aforementioned refactor in #1676. Should we keep logging as there are errors thrown there?

Reply:
I think that debug-level logging at least makes sense, but I'd leave this up to you. I've included some logging-related comments in the PR review though because we are also instantiating errors that get hidden in the catch block. What is your take on this?

Reply:
I agree - removing this here is pre-emptive. I reverted the removal in a3e9d98e906fac4a548304dbcba9544e68264764.

I'll adjust it in my proposed refactor instead (where logging will remain but in a simplified wrapper).